Makefile

```diff
@@ -0,0 +1,99 @@
+K8S_DIR := .k8s
+K8S_CLUSTER_NAME := storj
+KUBECONFIG = $(realpath $(K8S_DIR)/config)
+BUCKET_NAME ?= velero
+STORJ_ACCESS ?= # Required
+DOCKER_IMAGE := storjthirdparty/velero-plugin
+DOCKER_TAG ?= dev
+VELERO_BACKUP_LOCATION := default
+
+export KUBECONFIG
+
+# check minimun required tools
+$(if $(shell which kind),,$(error kind tool is required))
+$(if $(shell which velero),,$(error velero tool is required))
```

Comment:
These checks do not work on my side. `kind` is in my path, but it does not seem to find it (everything works fine if I comment those lines).
I'm not familiar with this syntax, but if I do something like:
`$(if $(shell which kubectl),$(error it is fine)),$(error kubectl tool is required))`
I do get the error "it is fine".

Reply:
:thinking: probably it's due to some differences between `make` versions.
I will remove them.